src/app/lib/config/services/vietnamese.js

```diff
@@ -249,31 +249,27 @@ export const service = {
       },
       {
         title: 'Việt Nam',
-        url: '/vietnamese/vietnam',
+        url: '/vietnamese/topics/ckdxnx1x5rnt',
       },
       {
         title: 'Thế giới',
-        url: '/vietnamese/world',
+        url: '/vietnamese/topics/cnlv9j1ekq0t',
       },
       {
         title: 'Diễn đàn',
-        url: '/vietnamese/forum',
+        url: '/vietnamese/-54540875',
```

Comment:
@jordit This one doesn't look right - is it missing something, or does it need the dash taken out